core/base/inc/Bswapcpy.h

```diff
@@ -30,7 +30,7 @@
 //    n - is a number of array elements to be copied and byteswapped.   //
 //        (It is not the number of bytes!)                              //
 //                                                                      //
-// Note: It is note safe to call these routines with n == 0.            //
+// Note: It is not safe to call these routines with n == 0.            //
```

Comment:
Can you just align the `//` at the end please?

Reply:
Done.